package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
- [ ] `react-redux` and should be present in `package.json`. Will be needed in order to use Redux in the next millstones/projects